winbuild/Makefile.vc

```diff
@@ -235,3 +247,7 @@ $(MODE):
 copy_from_lib:
 	echo copying .c...
 	FOR %%i IN ($(CURLX_CFILES:/=\)) DO copy %%i ..\src\
+!ELSE
+clean:
+	@$(MAKE) /NOLOGO /F MakefileBuild.vc clean mode=clean
```

Comment:
I love the with_XYZ_DEVEL, but it has one draw back in that AFAICS the windows zlib build doesn't have an INSTALL, so I (any everyone wanting to use it) would need to build four pseudo installations (debug,release)x(x86,x63) before I did a build.

Actually & IIRC, if zlib had had an `install ` target I would have probably not even started down the route and 'just' installed OpenSSL and ZLIB into the build tree and done with `SSL_LIBS`, `ZLIB_LIBS `and `WITH_DEVEL`

Sorry for the brevity =  I'll be able to respond in more detail over the weekend.